commands/destroy.js

```diff
@@ -0,0 +1,35 @@
+const cli = require('heroku-cli-util')
+
+let usage = `
+    ${cli.color.bold.underline.magenta('Usage:')}
+    ${ cli.color.cmd('heroku container:destroy web')} # Destroys the web container`
+
+module.exports = function (topic) {
+  return {
+    topic: topic,
+    command: 'destroy',
```

Comment:
I would use `rm` for consistency with the Docker CLI. This matches the other commands.

Reply:
Wfm. I named it destroy as it's how `apps:destroy` is named. But following the docker cli makes more sense.